src/PokemonList.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react'
+
+export default class PokemonList extends Component {
+    render() {
+        return (
+            <div className="pokemon-list">
+                <img 
+                    alt={this.props.pokemon.url_image}
```

Comment:
The alt might be clearer for blind folks if it's the name as opposed to the url.

Reply:
Ill fix that up good point!